.eslintrc.json

```diff
@@ -15,6 +15,17 @@
       "import/no-extraneous-dependencies": [
         "error", { "devDependencies": true }
       ],
+      "no-plusplus": [
+        "error", { "allowForLoopAfterthoughts" : true }
```

Comment:
OK. In moment-opening-times there were just a couple of loops so the change was not a big deal and I therefore switched to +=/-= but it does mean that .eslintrc has diverged between the two repos. Just noting it here for future reference.